scripts/build-element.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs-extra');
```

Comment:
This script doesn't work correctly, it must output ESM script + ESM browser script + UMD script. Look how it is done in build-js